Documentation/migration-scripts/mentoring/migrate-org-roles.sh

```
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Load environment variables from db.env
+if [ -f "./migration.env" ]; then
+    export $(grep -v '^#' ./migration.env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)